scripts/test.sh

```diff
@@ -4,4 +4,11 @@ python3 setup.py sdist --formats=gztar
 pip install dist/model-0.1.tar.gz
 
 python3 -m model.training --dataset_file=dataset.jsonl --model_output_directory=. --num_epochs=2
-pip install pytest && pytest tests/
+pip install pytest && pytest tests/test_file_exists.py
+
+rm labels.txt
+rm detection.tflite
+
+python3 -m model.training --dataset_file=dataset.jsonl --model_output_directory=. --num_epochs=2 --labels=green_square/blue_star
```

Comment:
I don't think we should actually need to execute the code in order to test the args parsing. Instead, we should probably refactor parse_args to take in args, instead of relying on the implicit use the default [sys.argv](https://docs.python.org/3/library/sys.html#sys.argv). The docs describe this in a little bit more detail [here](https://docs.python.org/3/library/argparse.html#parsing-arguments). Then, in the main, we can call `parse_args(sys.argv[1:])`. In the tests, we would just explicitly check that the labels are being parsed as expected & nothing else.